main.go

```diff
@@ -82,7 +92,13 @@ func main() {
 		"print the verified provenance to std out")
 	flag.Parse()
 
-	if provenancePath == "" || artifactPath == "" || source == "" {
+	if (provenancePath == "" || artifactPath == "") && ociImageRef == "" {
```

Comment:
I feel like we should have different commands for containers vs. artifacts

The combinations of how flags are used isn't really very user friendly and I feel like it will just get worse over time.

Reply:
...not that I expect that to be implemented in this PR. Just a general comment.

Reply:
For v2, I'll try to write a one-pager on the SLSA verifier API. There are a few issues on this (https://github.com/slsa-framework/slsa-verifier/issues/180) and I am worried without writing it up first I'll get it wrong.

Reply:
Just to be clear, I was simply thinking something like `slsa-verifier verify-artifact ...` and `slsa-verifier verify-image ...` something like that. Right now it's down to the combination of what flags users use. Commands allow us to have different flags for different CLI commands.

Reply:
Ah, yes I agree too we should have sub-commands separating the two.

Reply:
SGTM